File: Pagina_web/Entity/Inmueble.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Entity
{

    public class Inmueble
    {
        [Key]
        [Column(TypeName = "nvarchar(11)")]
        public String codigo { get; set; }
        [Column(TypeName = "nvarchar(100)")]
        public String Nombre { get; set; }  
        [Column(TypeName = "nvarchar(50)")]
        public String Estado { get; set; }
        [Column(TypeName = "nvarchar(100)")]
        public String direccion { get; set; }
        [Column(TypeName = "nvarchar(500)")]
        public String Descripcion { get; set; }
        [ForeignKey("CodInmueble")]
        public List<fotoInmueble> fotos { get; set; }
    }
}

File: Leco2/app/src/main/java/com/naacdeveloper/leco/modelos/Inmueble.kt
package com.naacdeveloper.leco.modelos

class Inmueble(codigo: String, nombre: String, estado: String, direccion: String, descripcion: String, fotos: ArrayList<FotoInmueble>?) {
    var codigo: String = "";
    var nombre: String = "";
    var estado: String = "";
    var direccion: String = "";
    var descripcion: String = "";
    var fotos: ArrayList<FotoInmueble>? = null;

    init{
        this.codigo = codigo;
        this.nombre = nombre;
        this.descripcion = descripcion;
        this.direccion = direccion;
        this.estado = estado;
        this.fotos = fotos;
    }
}